parallel-install/README.md

```diff
@@ -47,3 +47,51 @@ Once you have a configured `Deployment` instance, use the following functions ac
 ### Example
 
 To learn how to use the library to deploy Kyma on a Gardener cluster, see this [example](../parallel-install/example/example.go).
+
+## Utility Packages
+
+### Download Package
+`download` package allows you to download remote files.
+
+`download.GetFile` downloads a single file and returns the path to the downloaded file. Destination directory will be created if it does not exist.  
+Required parameters:
```

Comment:
```suggestion
The required parameters are:
```

Reply:
Please join this phrase with the previous paragraph. Apply in other places as well.